.README.md

```diff
@@ -0,0 +1,36 @@
+## {{ (datasource "git").name }} [![Build Status](https://travis-ci.org/cloudposse/terraform-aws-cloudwatch-logs.svg?branch=master)](https://travis-ci.org/cloudposse/terraform-aws-cloudwatch-logs)
+{{ (datasource "section").warning }}
+
+Terraform module for creation streams and group for them.
```

Comment:
Terraform module for creation of CloudWatch Log Streams and Log Groups for use with Fluentd.